test/integration/mapping_validation_test.rb

```diff
@@ -149,7 +149,8 @@ def setup
     mapping_rule_files.each do |file|
       raw_mappings = @sys.parse_yaml(file)
       [raw_mappings["input_taxonomy"], raw_mappings["output_taxonomy"]].each do |taxonomy_version|
-        next if !taxonomy_version.include?("shopify") || taxonomy_version.include?("shopify/2022-02")
+        next if !taxonomy_version.include?("shopify") || taxonomy_version.include?("shopify/2022-02") ||
+          taxonomy_version.include?("shopify/2024-07")
```

Comment:
Let's get a better test here

This will allow `shopify/2024-07` to be used as the input/output version of any mapping, which is not what we want. For e.g. in the `google/2021-09-21` mapping (from shopify) `shopify/2024-07` is not a valid version.

> Ok to do this in a follow up PR so we can unblock https://github.com/Shopify/product-taxonomy/pull/356 and https://github.com/Shopify/product-taxonomy/pull/229